lib/toy/object_instance.rb

```diff
@@ -56,6 +56,10 @@ def method(selector)
         # Check mixed-in Modules to see if they define the selector
         superclass.included_modules.each do |mixin|
           return Method.new(mixin) if mixin.instance_methods.include?(selector)
+
+          mixin.included_modules.each do |moar_mixins|
+            return Method.new(moar_mixins) if moar_mixins.instance_methods.include?(selector)
```

Comment:
Yeah, this is something I'm still struggling with 😭 Taking a TDD approach and trying to write the least amount of code to make the test pass vs just going for the code I know is "correct" in the end. It's been ages since I wrote this so I don't quite remember where my head was it -- I think I was trying not to get distracted by the complexity of traversing the mixin tree yet, and wanted to stick with a simple test case.